google_nest_sdm/webrtc_util.py

```diff
@@ -1,114 +1,108 @@
-# SDP Direction Constants
-DIRECTION_SENDRECV = 'sendrecv'
-DIRECTION_SENDONLY = 'sendonly'
-DIRECTION_RECVONLY = 'recvonly'
-DIRECTION_INACTIVE = 'inactive'
+"""Library with functions for manipulating WebRTC requests/responses."""
+from enum import StrEnum
```

Comment:
Formatting in this file will fail lint checks. You can the pre-commit and it'll auto fix everything.